pkg/reconciler/contour/lister.go

```diff
@@ -43,7 +43,20 @@ var _ status.ProbeTargetLister = (*lister)(nil)
 func (l *lister) ListProbeTargets(ctx context.Context, ing *v1alpha1.Ingress) ([]status.ProbeTarget, error) {
 	var results []status.ProbeTarget
 
-	visibilityKeys := config.FromContext(ctx).Contour.VisibilityKeys
+	cfg := config.FromContext(ctx)
+
+	var port int32
+	var scheme string
+	switch cfg.Network.HTTPProtocol {
+	case network.HTTPEnabled:
+		port, scheme = 80, "http"
+	case network.HTTPRedirected:
+		port, scheme = 443, "https"
+	case network.HTTPDisabled:
+		return results, nil
```

Comment:
This should be same as Redirected I think.

Reply:
Done.